### ISSUE TITLE
no-empty-file should be configurable

### ISSUE THREAD

**Issue by tkrotoff** [sindresorhus/eslint-plugin-unicorn#2218]
`no-empty-file` isn't configurable

A file with comments but without code is considered empty. More finesse would be appreciated.

Example: I would like to disallow empty files in general, but I would like to allow *.test.ts(x) files to contain only comments

Use case:

```TypeScript
// File: useScreenSizeHint.test.ts
// No need to write tests here, useScreenSizeHint.ts source code is already tested by ScreenSizeHintProvider.test.tsx
```

ESLint configuration:
```TypeScript
rules: {
  'unicorn/no-empty-file': 'error'
},
overrides: [
  {
    files: ['**/*.test.ts', '**/*.test.tsx'],
    rules: {
      // Empty test files with comments are legit
      'unicorn/no-empty-file': ['error', { allow: ['comments'] }]
    },
  }
]
```

**Comment by dimaMachina** [comment]
Just use eslint-disable comment for your particular case

**Comment by sindresorhus** [comment]
I agree that we should have an option to allow comments.

**Comment by AekoArray** [comment]
Hello, this is my first experience in open source. Could you see the implementation?
